_posts/2020-10-16-changelog-133.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #133"
+date: 2020-10-19 12:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5189 -->
+* Support configuring workspace features in `Cargo.toml`.
+  Now it is possible to enable or disable any specific workspace [feature](https://doc.rust-lang.org/cargo/reference/features.html) in the `Cargo.toml` file, and our name resolution and other code insight will take them into account.
```

Comment:
```suggestion
  Now it is possible to enable or disable any specific workspace [feature](https://doc.rust-lang.org/cargo/reference/features.html) in the `Cargo.toml` file, and plugin name resolution and other code insight features will take them into account.
```

Reply:
Maybe just even `Go to definition` instead of `plugin name resolution`?